bonito/models/configs/dna_r9.4.1@v3.1.toml

```diff
@@ -34,6 +34,7 @@ heads = 6
 attn_dropout = 0.1
 ff_dropout = 0.1
 use_self_attn = false
+num_encoder_layers_attend = 6
```

Comment:
6 because it also includes the fmap right after the conv encoder (and before the first RNN)